src/screens/SettingsScreen.js

```diff
@@ -153,15 +167,26 @@ function clearData(store) {
 function generateID(store) {
   const alphabet = ['A','B','C','D','E','F','G','H','I','J','K','L','M',
                     'N','O','P','Q','R','S','T','U','V','W','X','Y','Z'];
-  const min = 10;
-  const max = 99;
+  const min = 100;
+  const max = 999;
+
+  if (store.get('testBool') == 0) {
+    const num = String(Math.round(min + Math.random() * (max - min)));
+    const letter1 = alphabet[Math.floor(Math.random()*alphabet.length)];
+    const letter2 = alphabet[Math.floor(Math.random()*alphabet.length)];
+    const letter3 = alphabet[Math.floor(Math.random()*alphabet.length)];
 
-  const num = String(Math.round(min + Math.random() * (max - min)));
-  const letter = alphabet[Math.floor(Math.random()*alphabet.length)];
+    const id = letter1+letter2+letter3+num;
```

Comment:
did you run flow before committing your changes? i suspect it might want you to explicitly cast number to string here (and in other places).